pallets/fee-lock/src/benchmarking.rs

```diff
@@ -105,7 +105,7 @@ benchmarks! {
 		#[cfg(not(test))]
 		assert_ok!(<T as Config>::Xyk::create_pool(caller, valuating_token_id, pool_amount, token_id, pool_amount.saturating_mul(2u8.into())));
 
-		let value: BalanceOf<T> = 1980_u32.into();
+		let value: BalanceOf<T> = 2000_u32.into();
```

Comment:
does it meat that some rounding has change? that will explode in e2e i guess ..

Reply:
valuation changed to current state formula vs expected swap formula, so the test value had to change too

previously we had the valuation in fee lock pallet, then it was cleaned up and used a trait from market pallet that relays further to XYK
but the XYK method was using calculate_sell_price_no_fee which takes into account the expected swap amount
there is a fix to replace that method with actual state value as stated in docs 
check change in XYK pallet

yes, unfortunately e2e will have to adapt, but only for cases related to valuation